src/components/Pages/home/Doctor.js

```diff
@@ -0,0 +1,95 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import React from "react";
+
+const DoctorCtn = styled.div`
+  width: 350px;
+  height: 350px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  transition: all 100ms ease-in 0ms;
+
+  .img-ctn {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 350px;
+    height: 350px;
+  }
+
+  .doc-img {
+    width: 150px;
+    height: 150px;
+    margin: auto;
+    border-radius: 100%;
+  }
+
+  .img-bg {
+    width: 150px;
+    height: 150px;
+    margin: auto;
+    border-radius: 100%;
+    background: var(--img-background);
+  }
+
+  .doc-name {
+    text-align: center;
+    font-family: "Ubuntu Light";
+    margin: 10px 0 0 0;
+    font-size: 1.2rem;
+  }
+
+  .doc-speciality {
+    text-align: center;
+    font-family: "Ubuntu Thin";
+    margin: 20px 0 0 0;
+    color: var(--text-black);
+  }
+
+  .doc-cost {
+    text-align: center;
+    font-family: "Ubuntu Thin";
+    margin: 10px 0 0 0;
+    color: var(--text-black);
+  }
+
+  :hover {
+    cursor: pointer;
+    background-color: rgba(227, 228, 227, 0.51);
+    border-radius: 10px;
+  }
+`;
+
```

Comment:
Try to separate style from components. Make changes using classname. Makes code cleaner